src/main/java/com/task10/crudapi_login/service/ClientService.java

```diff
@@ -9,4 +9,5 @@ public interface ClientService {
 
     Client findById(int id);
 
+    Client createClient(Client client);
```

Comment:
ここも細かいですが、Client Serviceの関数なので、createという関数名の方が意味の重複がなくて良さそうです！

```suggestion
    Client create(Client client);
```

Reply:
修正いたしました。
fef6a6aed9a988a40cf628d19370dfb070faa790